### ISSUE TITLE
Deprecate the separate forum repository

### ISSUE THREAD

**Issue by waciumawanjohi** [ossu/computer-science#551]
The forum was created about a year ago: https://github.com/ossu/forum

However, participation has been sparse. In 2018, only 6 issues were opened. Of these, only 2 had a member of the community attempt to answer the questions raised.

I propose that the advice in the forum readme go into the Extras of Computer Science, the link to the forum be removed from the CS readme, and the forum repository deleted.

**Comment by hanjiexi** [comment]
Yeah, I agree. I think users have found it confusing to have a separate repo for the forum. For example, I see some users opened issues in the forum repo that were only relevant to CS.

It's better for the Issues board of each repo to act as the forum for discussions related to the business of that repo. Thus, if people want to talk about the future of OSSU as a whole, [ossu/roadmap](https://github.com/ossu/roadmap) would be the best place.

**Comment by waciumawanjohi** [comment]
Ok. The forum has been archived for weeks and no one has noticed or complained. I spent a fair amount of time ensuring that the page and all its issue pages were indexed on archive.org:
https://web.archive.org/web/*/https://github.com/ossu/forum

OSSU's account does not allow making repos private, so I have moved the repository to my account and will make it private.  This is in lieu of deleting it immediately. After 2 months if there are no objections I will delete the repository outright.
